libs/barista-components/sunburst-chart/src/sunburst-chart.ts

```diff
@@ -105,36 +137,14 @@ export class DtSunburst implements OnChanges {
   constructor(
     private _overlayService: Overlay,
     private _viewContainerRef: ViewContainerRef,
+    private _platform: Platform,
     private _elementRef?: ElementRef<HTMLElement>,
     @Optional()
     @Inject(DT_UI_TEST_CONFIG)
     private _config?: DtUiTestConfiguration,
   ) {}
 
-  ngOnChanges(changes: SimpleChanges): void {
-    if (changes.series?.currentValue !== changes.series?.previousValue) {
-      this.filledSeries = fillNodes(this.series);
-      this.render();
-    }
-
-    if (changes.selected?.currentValue !== changes.selected?.previousValue) {
-      this._selected = getSelectedNodesFromOutside(
-        this.filledSeries,
-        this.selected,
-      );
-      this.render();
-    }
-
-    if (
-      changes.valueDisplayMode?.currentValue !==
-      changes.valueDisplayMode?.previousValue
-    ) {
-      this.labelAsAbsolute =
-        this.valueDisplayMode === DtSunburstValueMode.ABSOLUTE;
-    }
-  }
-
-  select(event?: MouseEvent, slice?: DtSunburstSlice): void {
+  select(event?: MouseEvent, slice?: DtSunburstChartSlice): void {
```

Comment:
This needs to be internal - this is not a consumer facing API :) 
```suggestion
  _select(event?: MouseEvent, slice?: DtSunburstChartSlice): void {
```